nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/dto/status/ProcessGroupStatusSnapshotDTO.java

```diff
@@ -81,6 +81,9 @@ public class ProcessGroupStatusSnapshotDTO implements Cloneable {
     private Integer terminatedThreadCount = 0;
     private Long processingNanos = 0L;
 
+    @Schema(description = "Represents the processing performance for all the processors in the given process group.")
+    private ProcessingPerformanceStatusDTO processingPerformanceStatusDTO;
```

Comment:
One important thing to note, the member variables and get/set methods should not include `DTO`, even though the class name itself should have `DTO`.